app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
-  belongs_to :user
   belongs_to :post
+  belongs_to :user
+
+  after_create :update_likes_counter
+
```

Comment:
- OPTIONAL
You could've used the `after_save` helper method as you did in the comment model for more consistent logic. 🙏